src/main/java/org/walkmod/WalkModFacade.java

```diff
@@ -219,6 +223,58 @@ public WalkModFacade() {
 		this(false, true, false);
 	}
 
+
+	/**
+	 * Initalizes a Walkmod service
+	 *
+	 * @param walkmodCfg
+	 *            Walkmod configuration file. If null, file named 'walkmod.xml'
+	 *            is searched at the root.
+	 * @param options
+	 *            Map of option. See {@link Options} and {@link OptionsBuilder}
+	 *            for available options and default values.
+	 * @param configurationProvider
+	 *            Configuration provider responsible for the resolution of
+	 *            plugins (used to use custom classloading strategies).
+	 *            If null Ivy is used.
+	 */
+	public WalkModFacade(File walkmodCfg, Map<String,Object> options, ConfigurationProvider configurationProvider) {

```

Comment:
> I would prefer to have a Map constructor into the OptionsBuilder and restrict the WalkmodFacade parameter to OptionsBuilder. Could you conserve a similar syntax sugar in Groovy?

Agree. Even tough I feel using Options is the right answer from a desing stand point, using OptionsBuilder points developer in the right direction making the API a lot easier to use.
Since adding the Map constructor provides the same syntax options I'll apply it :+1: